### DISCUSSION TITLE
Any plans to continue this project?

### DISCUSSION THREAD

**Discussion by iMohamedSamir** [troublediehard/magento-react-native-community#11] [General]
Hello @dimaportenko Idk how far this project is gone, I tried the previous REST version and it was great but I got caught up in something and forgot about it the past few years, I knew back than you was planning on going graphql which I was excited for and I think you have finished it?
Anyway, any plans on continue improving this version?

**Comment by dimaportenko** [comment]
hi @iMohamedSamir. As you can see this project is stale for a year now. At the moment there are no plans to continue work on it.
